### DISCUSSION TITLE
useEnsAddress hook not working

### DISCUSSION THREAD

**Discussion by adeolu98** [wevm/wagmi#2164] [Q&A]
It appears useEnsAddress and useEnsName hooks on  wagmi v0.12.0 are not working as expected right now. Anyone else having the same issue?

**Comment by tmm** [comment]
Seems to work fine for me. Do you have a minimal reproducible example?

**Reply by adeolu98** [reply]
```
  const { data } = useEnsName({
    address: contractAddr as `0x`,
  });

  const { data } = useEnsAddress({
    name: ensName,
    chainId: 1
  });

```

This is how i use it in my app. For context the app i refer to is my project, [https://www.textmeanon.com/](https://www.textmeanon.com/). Please if you could check it out and try to use it to see what im talking about. Ens look up and name resolving were working up until yesterday with no new changes pushed by me. Even the connect wallet hook from rainbow wallet ui kit which uses wagmi too doesnt show my ens name anymore when i connect with my ens address. 

![1B341189-0E90-4434-B553-82B29F764FC8_1_201_a](https://user-images.githubusercontent.com/39372980/231242997-d6d3d0a8-6359-4802-91cc-a18ab4781973.jpeg)

The connected addr in image is registered on ens as adeoluwa.eth so it should automatically be shown like its always done in the past.

**Reply by tmm** [reply]
What address are you connected with for RainbowKit? Your ENS reverse record might not be configured correctly. 

Everything is working in [this StackBlitz](https://stackblitz.com/edit/new-wagmi-jsmefz?file=src/App.tsx) if you want to try it out.

**Reply by adeolu98** [reply]
Youre right, it is working in the link you shared, I guess i have to look at my code more
